test/metrics/npath/test_all_types.py

```diff
@@ -191,15 +191,7 @@ def test_complex_if_else_with_npath_complexity_of_5(self) -> None:
         ).strip()
         assert self._value(content) == 5
 
-    @pytest.mark.xfail(
```

Comment:
@KachanovYev it fixes #809 as well! Thank you!
Can you reference it in the PR description too?